S/SoapyPlutoSDR/build_tarballs.jl

```diff
@@ -7,46 +7,33 @@ version = v"0.2.1" # not yet tagged
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/pothosware/SoapyPlutoSDR.git", "ac9a9da5c14c73e752796618d56e259ca1ac6b11")
+    GitSource("https://github.com/pothosware/SoapyPlutoSDR.git", "a07c37230369653818b3a5c448c00cee1ac9f8e5")
 ]
 
 dependencies = [
-    Dependency("libiio_jll"; compat="~0.23.0"),
-    Dependency("soapysdr_jll"; compat="~0.8.0"),
-    Dependency("libad9361_iio_jll"; compat="~0.2.0")    
+    Dependency("libiio_jll"; compat="~0.24.0"),
+    Dependency("soapysdr_jll"; compat="~0.8.0")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd SoapyPlutoSDR
 
 mkdir build && cd build
-if [[ "${target}" != *-apple-* ]]; then
-    cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
-        -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
-        -DCMAKE_BUILD_TYPE=Release \
-        ..
-    make -j${nproc}
-    make install
-fi
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    ..
+make -j${nproc}
+make install
 if [[ "${target}" == *-apple-* ]]; then
-    cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
-        -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
-        -DCMAKE_BUILD_TYPE=Release \
-        -DLibIIO_INCLUDE_DIR=${libdir}/iio.framework/Versions/0.23/Headers/ \
-        -DLibIIO_LIBRARY=${libdir}/iio.framework/Versions/0.23/iio \
-        -DLibAD9361_INCLUDE_DIR=${libdir}/ad9361.framework/Versions/0.2/Headers/ \
-        -DLibAD9361_LIBRARY=${libdir}/ad9361.framework/Versions/0.2/ad9361 \
-        ..
-    make -j${nproc}
-    make install
     mv ${libdir}/SoapySDR/modules0.8/libPlutoSDRSupport.so  ${libdir}/SoapySDR/modules0.8/libPlutoSDRSupport.dylib
 fi
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = supported_platforms(;experimental=true)
+platforms = supported_platforms()
```

Comment:
Why did you take away `experimental=true`?

Reply:
It was required for aarch64-apple-darwin, but it is now included in the supported_platforms. I guess keeping it is okay? It currently does not add any platforms at the moment.